cabal.project

```diff
@@ -64,3 +64,75 @@ source-repository-package
   type: git
   location: https://github.com/shmish111/github.git
   tag: cc27b9de4d5d0939235fa9a8b418de3ea4807bab
+
+-- Node protocols
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-base
+  tag: 80deee31f8d9422b8e090e55b17e1a714153180b
+  subdir: binary
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-base
+  tag: 80deee31f8d9422b8e090e55b17e1a714153180b
+  subdir: slotting
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-prelude
+  tag: 00487726c4bc21b4744e59d913334ebfeac7d68e
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-prelude
+  tag: 00487726c4bc21b4744e59d913334ebfeac7d68e
+  subdir: test
```

Comment:
Do we actually need this?

Reply:
We need it for the definition of `NoUnexpectedThunks`.